R/DBConnection.R

```diff
@@ -106,6 +106,41 @@ setGeneric("dbSendQuery",
   valueClass = "DBIResult"
 )
 
+#' @export
+dbBreak <- structure(list(), class = "dbiAbort")
```

Comment:
I'm not sure that was the intention behind this... This seems like a discussion worth having. The idea was for `dbBreak` to be analogous to `break`: i.e. a basic-looking control statement that allows you to get out of the `while` loop inside `dbFetchChunkedQuery`. It would be great, of course, to be able to simply use `break` here, but that doesn't work. It still sucks that you have to actually return `dbBreak`, rather than just invoke it; but oftentimes that will turn out to look exactly the same. Even when it doesn't, that construct still feels more intuitive to me from a user perspective than having to call a function to break out of the loop... So, even though we use `tryCatch` logic to catch this, it really shouldn't be thought of as a condition object, but as a control statement (or, more accurately, a sentinel value that mimics a control statement)...

That being said, I understand that this may just be confusing. @jcheng5, @hadley , thoughts?